pkg/persistence/object_store.go

```diff
@@ -276,6 +281,30 @@ func (s *objectBackupStore) GetDownloadURL(backup string, target arkv1api.Downlo
 	}
 }
 
+func (s *objectBackupStore) GetRevision() (string, error) {
+	rdr, err := s.objectStore.GetObject(s.bucket, s.prefix+"revision")
```

Comment:
Would it make sense to move `s.prefix+"revision"` into a `getRevisionKey` function?

Reply:
yeah, makes sense